### PR TITLE
update CHANGELOG.md

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,126 +1,22 @@
...
-  - Rewrote help strings in `woof.cfg` (thanks to @MrAlaux).
-  - Added interpolation of wipe melt animation (from Rum and Raisin Doom).
-  - Fit finale text to widescreen instead of inserting line breaks.
-  - Always use traditional menu ordering (removed Boom choice).
-  - Moved all autoload and internal resources to `woof.pk3` archive.
-  - Allowed setting `gamedescription` (and thus the window title) with DEHACKED.
-  - Allowed up to 10 episodes in UMAPINFO.
-  - Added more paths to find IWADs.
-  - Removed source port credits screen, restoring the vanilla demo loop.
+* Process TRAKINFO in IWADs. Compatibility with Chex Quest 3: Vanilla Edition 1.0.
```
Comment (MelodicSpaceship) [inline CHANGELOG.md:3]: Chex 3 Vanilla 1.0 doesn't actually have a TRAKINFO in the IWAD, the digital TRAKINFO music is actually a separate WAD to be loaded on top of the IWAD.

Comment (rfomin) [reply CHANGELOG.md]: Fixed in 107ad46

@elf-alchemist send me a version with TRAKINFO, it's Bethesda.net only?

Comment (elf-alchemist) [reply CHANGELOG.md]: Hi, yes it was part of the testing I was doing for the BNet version that I happened to come across that it wasn't loading the `TRAKINFO` in an IWAD (I thought it'd be a shame to entirely exclude all possible custom IWADs in the future from having it built-in) but also the fact that I tend to merge and customize the PWADs in my local setup for a more concise package :P

**Review by fabiangreffrath — APPROVED** [review]
Fine with me. Release 15.0.1 now?

**Comment by rfomin** [comment]
> Fine with me. Release 15.0.1 now?

Yes, please. Should we merge #2078?

**Comment by fabiangreffrath** [comment]
> Yes, please. Should we merge #2078?

If you are confident with this approach, please do.

**Pull request merged**